components/ReviewCard.tsx

```diff
@@ -74,16 +76,38 @@ export const DiffView: React.FC<DiffViewProps> = ({ diff = '' }) => {
 }
 
 const MemoDiffView = memo(DiffView)
-
+type UserProfileProps = {
+  username: string | undefined | null
+  name: string | undefined | null
+}
+const UserProfile: React.FC<UserProfileProps> = ({ username, name }) => {
```

Comment:
Rename this to something else to make it clear that it renders the reviewer user.

`UserProfile` makes me think that it's something from the profile page.

Reply:
Renamed to `ReviewerProfile`.